package.json

```diff
@@ -83,34 +83,37 @@
     "@storybook/preset-typescript": "^3.0.0",
     "@storybook/react": "^5.3.18",
     "@types/classnames": "^2.2.10",
+    "@types/enzyme": "^3.10.5",
     "@types/jest": "^25.1.4",
+    "@types/node": "^14.0.4",
     "@types/react": "^16.9.27",
-    "@typescript-eslint/eslint-plugin": "^2.33.0",
+    "@types/react-dom": "^16.9.8",
+    "@typescript-eslint/eslint-plugin": "^2.26.0",
     "@typescript-eslint/parser": "^2.26.0",
     "auto-changelog": "^2.0.0",
-    "babel-eslint": "^10.1.0",
+    "babel-eslint": "^10.0.3",
     "babel-loader": "^8.1.0",
     "babel-plugin-external-helpers": "^6.22.0",
     "babel-plugin-transform-es2017-object-entries": "^0.0.5",
     "cross-env": "^7.0.2",
     "cssnano": "^4.1.10",
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.13.0",
-    "eslint": "^7.0.0",
+    "eslint": "^6.8.0",
```

Comment:
I assume this downgrade was connected to the babel-eslint one?

Reply:
No, this was actually connected to `react-scripts`. Kind of disappointing, but it's dependency is on `^6.6.0` 😢